### DISCUSSION TITLE
Smart management based on tags / labels, domains, states, eg... for devices, entities, scripts and automations.

### DISCUSSION THREAD

**Discussion by 10bn** [home-assistant/frontend#8683] [Accessibility]
This is a proposal for a new management approach on based on tags / labels, domains, states, eg... for devices, entities, scripts and automations. 
The idea is inspired by how easily you can organize spreadsheets or create smartplaylists in itunes. 

Smart folder / search rules for Devices / Entities, Scripts, Automations:

For beginners the current setup might be fine but after while when you dive deeper into the Home Automation jungle and the more automations or scripts one will most likely accumulate it gets very messy. Therefore it would be nice if we could tag or lable entities, scripts, automations, eg.... and create smart search rules displayed as folders to organize things accurately. 

Tags / Labels or groups:

Currently devices or entities area attribute can only be assigned once peer device. Ok though, we can simply use groups if we want devices to be in multiple groups, but still why do we need groups & areas if this could be organized by using tags / labels? 

If you like the idea please leave your feedback or suggestions about this.

**Comment by spacegaier** [comment]
Related ticket: https://github.com/home-assistant/frontend/issues/4542

**Comment by Misiu** [comment]
The advantage of tags is the ability to add more than one. For example, you could add the Living room and Light tag to automation, this way when filtering you will be able to filter all automations for the living room or all automations that interact with lights.
Folders can't do that.

**Reply by 10bn** [reply]
exactly!

**Comment by h4de5** [comment]
I think this was done in: https://www.home-assistant.io/blog/2021/04/07/release-20214/#filtering-automations-scripts-and-scenes
